apps/table-frontend-e2e/src/support/commands.ts

```diff
@@ -16,7 +16,7 @@ declare namespace Cypress {
 //
 // -- This is a parent command --
 Cypress.Commands.add('login', (email, password) => {
-  console.log('Custom command example: Login', email, password);
+  console.info('Custom command example: Login', email, password);
```

Comment:
I'm guessing you don't want to be logging passwords here?

Reply:
Oh yeah, I did not even check the e2e directory as it was auto generated and I do not have e2e tests yet, but will remove!